modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/di/ExperimentModule.kt

```diff
@@ -0,0 +1,69 @@
+package au.com.shiftyjelly.pocketcasts.analytics.di
+
+import android.content.Context
+import au.com.shiftyjelly.pocketcasts.analytics.AccountStatusInfo
+import au.com.shiftyjelly.pocketcasts.analytics.BuildConfig
+import au.com.shiftyjelly.pocketcasts.analytics.experiments.Experiment
+import au.com.shiftyjelly.pocketcasts.analytics.experiments.ExperimentProvider
+import au.com.shiftyjelly.pocketcasts.servers.di.Cached
+import au.com.shiftyjelly.pocketcasts.utils.log.LogBuffer
+import com.automattic.android.experimentation.ExperimentLogger
+import com.automattic.android.experimentation.VariationsRepository
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import java.io.File
+import javax.inject.Singleton
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import okhttp3.OkHttpClient
+
+@Module
+@InstallIn(SingletonComponent::class)
+object ExperimentModule {
+
+    @Provides
+    @Singleton
+    fun provideExperimentProvider(
+        accountStatusInfo: AccountStatusInfo,
+        repository: VariationsRepository,
+    ): ExperimentProvider = ExperimentProvider(accountStatusInfo, repository)
+
+    @Provides
+    @Singleton
+    fun provideVariationsRepository(
+        @ApplicationContext context: Context,
+        @Cached okHttpClient: OkHttpClient,
+    ): VariationsRepository {
+        val directory = File(context.filesDir, "experiments").apply {
+            if (!exists()) mkdirs()
+        }
```

Comment:
```suggestion
        val directory = File(context.filesDir, "experiments")
```

The Tracks/ExPlat SDK will handle the case of missing directory now (just to double-check, it'd be great to verify if it's true on fresh app installation 🙏 )